tests/dialects/test_trino.py

```diff
@@ -0,0 +1,67 @@
+from tests.dialects.test_dialect import Validator
+
+
+class TestTrino(Validator):
+    dialect = "trino"
+
+    def test_DAY_OF_WEEK(self):
```

Comment:
put these in one function and call it

def test_extract and make sure to test identity, meaning test presto with presto